src/genegraph/transform/clinvar/clinical_assertion.clj

```diff
@@ -275,24 +275,28 @@
                       FILTER(?release_date <= ?max_release_date) }
                       order by desc(?release_date)
                       limit 1"
-                     {:trait_set_id trait-set-id
-                      :max_release_date release-date})]
-    (when (= 0 (count rs))
-      (log/error :fn :get-trait-by-id
-                 :msg "No matching trait set"
-                 :trait-set-id trait-set-id
-                 :release-date release-date))
-    (first rs)))
+                       {:trait_set_id trait-set-id
+                        :max_release_date release-date})]
+      (when (= 0 (count rs))
+        (log/error :fn :get-trait-set-by-id
+                   :msg "No matching trait set"
+                   :trait-set-id trait-set-id
+                   :release-date release-date))
+      (first rs)))
 
 (defn get-trait-set-by-version-of
   [^RDFResource trait-set-vof ^String release-date]
-  (log/info :fn :get-trait-set-by-id
-            :trait-set-vof trait-set-vof
-            :release-date release-date)
+  (log/debug :fn :get-trait-set-by-version-of
+             :trait-set-vof trait-set-vof
+             :release-date release-date)
+  (comment
+    "This resource type pattern was originally included in the following select."
+    "It was found to cost hundreds of milliseconds and was taken out. Assuming
+     dc/is-version-of is an identifier that can only have one of these types, the
+     query will perform successfully without it. If this assumption becomes no longer
+     true, it will need to be re-added"
+    "{ ?i a :vrs/Condition } union { ?i a :vrs/Disease } union { ?i a :vrs/Phenotype }")
```

Comment:
Add an assert?

Reply:
Makes sense to me, an assert would be cheap.

Reply:
And the error condition in that fn seems like it should be …
```clojure
(when-not (== 1 (count rs))
      (log/error :fn :get-trait-set-by-version-of
                 :msg "No matching trait set"
                 :trait-set-vof trait-set-vof
                 :release-date release-date))
```